pkg/providers/vsphere/vmlifecycle/update_status.go

```diff
@@ -385,11 +385,14 @@ func reconcileStatusStorage(
 	_ *object.VirtualMachine,
 	_ ReconcileStatusData) []error { //nolint:unparam
 
+	var errs []error
```

Comment:
nit: declar name for the return value directly?

Reply:
Agreed with @lubronzhan . Named return values are tricky. We should have probably used them in the validator (which is where the `var errs []error` pattern originated), but I agree we should stick with convention for now.